From eb5ecf0d0fdfc26d524f65eb92693f16b380878b Mon Sep 17 00:00:00 2001
From: xbw <xbw@rock-chips.com>
Date: Mon, 20 May 2013 12:15:40 +0800
Subject: [PATCH] Please update your board-xxx-sdmmc.c in your old project. For
 exmaple board-rk30-sdk-sdmmc.c.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

If you do not modify your files,you will encounter an error similar to the following.
      3066 4.2 develop-3.0的分支编译出错，报告
   drivers/built-in.o: In function rk29_sdmmc_set_ios
   bt_plat_smdk.c:(.text+0x16cfd8): undefined reference to rk29_sdmmc_gpio_open
   make: *** [.tmp_vmlinux1] 错误 1
---
 arch/arm/mach-rk30/board-rk30-sdk-sdmmc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-rk30/board-rk30-sdk-sdmmc.c b/arch/arm/mach-rk30/board-rk30-sdk-sdmmc.c
index e99918c8e635..f419cedb898a 100755
--- a/arch/arm/mach-rk30/board-rk30-sdk-sdmmc.c
+++ b/arch/arm/mach-rk30/board-rk30-sdk-sdmmc.c
@@ -1035,7 +1035,7 @@ static struct rksdmmc_gpio_board rksdmmc1_gpio_init = {
 *************************************************************************/
 
 #if !defined(CONFIG_SDMMC_RK29_OLD)	
-static void rk29_sdmmc_gpio_open(int device_id, int on)
+void rk29_sdmmc_gpio_open(int device_id, int on)
 {
     switch(device_id)
     {
-- 
2.35.3

